### ISSUE TITLE
[BUG]Chinese URL request is abnormal after passing through the gateway.

### ISSUE THREAD

**Issue by MarsOu1995** [apache/incubator-shenyu#1778]
## Describe the bug
In the browser, when Chinese is used as the url parameter, it will be encoded and sent to the gateway. But after passing through the gateway, what is passed to the backend becomes non-encoded Chinese.
I suspect the #1770  update caused the error .
## Environment

- ShenYu version(s): [e.g. 2.4.0]

## Steps to reproduce
browser: Get http://${gateway}/XXXX/api/manager?keyword=何铭健
The browser will automatically encode：Get http://${gateway}/XXXX/api/manager?keyword=%E4%BD%95%E9%93%AD%E5%81%A5

## Debug logs
<details>
<summary>backend error example</summary>

```
java.lang.IllegalArgumentException: Invalid character found in the request target [/api/manager?keyword=0xe60x9e0x970xe90x870x910xe60x800xa1 ]. The valid characters are defined in RFC 7230 and RFC 3986
```
</details>

Packets captured by the backend 
![image](https://user-images.githubusercontent.com/20200721/125881280-cd4506f2-96b3-4c1e-ba9e-8665476f764b.png)

**Comment by KevinClair** [comment]
Thanks for your report, can you fix it?

**Comment by MarsOu1995** [comment]
Yes, I will submit it later.

**Comment by KevinClair** [comment]
By the way, verify that errors can result when there are special characters in the parameter.

**Comment by SouthEastLJ** [comment]
is need to determine if the incoming request is already encoded?

**Comment by MarsOu1995** [comment]
> is need to determine if the incoming request is already encoded?

No, the browser will automatically encode.

**Comment by MarsOu1995** [comment]
bug fix #1786
